ui/src/common/components/DateRangeAggregation.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { DatePicker, Row } from 'antd';
+import AggregationBox from './AggregationBox';
+import {
+  DATE_RANGE_FORMAT,
+  RANGE_AGGREGATION_SELECTION_SEPARATOR,
+} from '../constants';
+
+function DateRangeAggregation({ name, onChange, selections }) {
+  const [startDate = '', endDate = ''] = useMemo(
+    () =>
+      selections ? selections.split(RANGE_AGGREGATION_SELECTION_SEPARATOR) : [],
```

Comment:
it may be better to default selections `selections = ''`, instead of this check.